fusor-ember-cli/app/templates/components/rhci-start.hbs

```diff
@@ -43,7 +43,7 @@
   </div>
 
   <div class='download-reqs'>
-    <a download href="QCI_Requirements.txt">
+    <a href="/assets/r/QCI_Requirements.txt" target="_blank">
```

Comment:
@eriknelson, @jwmatthews, which browser did `<a download` not work? Now instead of downloading, the browser just opens the file in any tab, right?